assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -3444,7 +3450,7 @@ public void updateEntityReferences(String toContext, Map<String, String> transve
                     nAssignment.setTypeOfGrade(oAssignment.getTypeOfGrade());
                     nAssignment.setTypeOfSubmission(oAssignment.getTypeOfSubmission());
                     // when importing, refer to property to determine draft status
-                    if (serverConfigurationService.getBoolean("import.importAsDraft", false)) {
+                    if (serverConfigurationService.getBoolean("import.importAsDraft", true)) {
```

Comment:
is this an opinionated change?

Reply:
According to the documentation this should've been true, and the 11 code was using reverse logic which is why it was set erroneously set to false.

Reply:
cool